### ISSUE TITLE
Clock covers the text when rotated sideways 

### ISSUE THREAD

**Issue by MoralCode** [DeveloperACE/MetricTime#4]
**What happened:**

```
Clock covers the text when rotated sideways 
```

**Version:** 1.0 (1)

**Operating system:** iOS 9.3.1

**Model:** iPhone 6

**Memory:** 97% (591 MB used out of 607 MB)

**Capacity:** 70% (38.7 GB used out of 55.6 GB)

**Locale:** en_US

**Network:** AT&T LTE (4G)

**WiFi:** Connected

**Logs:** 

![Screenshot](http://buglife.s3.amazonaws.com/reports/2971/attachments/original/929944a537397598c2cbf55c4a72c6863a41d494.png?1460575938)

**URL:** http://buglife.com/apps/metric-time/reports/1

Reported via Buglife

**Comment by MoralCode** [comment]
okay, so I am trying to add two constraints, one to horizontally center the clock, and one to make sure it stays some distance away from the top, however. I keep running into an issue where constraints cannot all be satisfied, and so it breaks the constraint that I just created:

```
2016-04-18 19:22:04.270 Metric Time[1519:447809] Unable to simultaneously satisfy constraints.
Probably at least one of the constraints in the following list is one you don't want. 
Try this: 
    (1) look at each constraint and try to figure out which you don't expect; 
    (2) find the code that added the unwanted constraint or constraints and fix it. 
(Note: If you're seeing NSAutoresizingMaskLayoutConstraints that you don't understand, refer to the documentation for the UIView property translatesAutoresizingMaskIntoConstraints)
(
"<NSAutoresizingMaskLayoutConstraint:0x15e670e70 h=--& v=--& Metric_Time.View:0x15e692e70.midX == + 115>",
"<NSLayoutConstraint:0x15e5048c0 Metric_Time.View:0x15e692e70.centerX == UIView:0x15e6953e0.centerX>",
"<NSAutoresizingMaskLayoutConstraint:0x15e55fd30 h=-&- v=-&- 'UIView-Encapsulated-Layout-Left' H:|-(0)-[UIView:0x15e6953e0]   (Names: '|':UIWindow:0x15e688ed0 )>",
"<NSLayoutConstraint:0x15e54e750 'UIView-Encapsulated-Layout-Width' H:[UIView:0x15e6953e0(320)]>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x15e5048c0 Metric_Time.View:0x15e692e70.centerX == UIView:0x15e6953e0.centerX>

Make a symbolic breakpoint at UIViewAlertForUnsatisfiableConstraints to catch this in the debugger.
The methods in the UIConstraintBasedLayoutDebugging category on UIView listed in <UIKit/UIView.h> may also be helpful.
```

I have looked through all my code and cannot find out where it is creating those other three conflicting constraints. Any help would be appreciated.

**Comment by MoralCode** [comment]
Updated screenshots for the latest commit c7c8e5af1e2756f7db7ff36106c827bc56cb5de4

![image](https://cloud.githubusercontent.com/assets/17362949/14626433/256d55a8-05a0-11e6-9193-425e00728fa6.png)
![image](https://cloud.githubusercontent.com/assets/17362949/14626434/2598aa96-05a0-11e6-951f-efc9312a1246.png)

**Comment by MoralCode** [comment]
Resolved: http://stackoverflow.com/a/36707830/3033386

**Comment by MoralCode** [comment]
issue fixed in commit 04c859ef74776e7d06100f00337b1c4575bc76c5

**Comment by MoralCode** [comment]
Updated screenshots:
![image](https://cloud.githubusercontent.com/assets/17362949/14641923/3f546a12-05fd-11e6-8116-ae520d5e3c5c.png)
![image](https://cloud.githubusercontent.com/assets/17362949/14641924/3f569684-05fd-11e6-8a5e-df575dad58ff.png)
